test/request_storage_test.exs

```diff
@@ -23,6 +23,19 @@ defmodule RequestStorageTest do
     {:ok, %{crawler: :test_spider}}
   end
 
+  test "Starting a worker twice doesn't break", context do
```

Comment:
could you adjust this test name to describe the expected underlying behaviour? "doesn't break" isn't very informative of what is being tested here.